gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/PegasusPlugin.java

```diff
@@ -1693,6 +1648,37 @@ protected GenerateDataTemplateTask configureDataTemplateGeneration(Project proje
     prepareLegacySchemasForPublishTask.dependsOn(destroyStaleFiles);
     dataTemplateJarDepends.add(prepareLegacySchemasForPublishTask);
 
+    // extension schema directory
+    File extensionSchemaDir = project.file(getExtensionSchemaPath(project, sourceSet));
+
+    if (!SharedFileUtils.getSuffixedFiles(project, extensionSchemaDir, PDL_FILE_SUFFIX).isEmpty())
```

Comment:
There is a hardcoded "extensions"  literal in the function `getExtensionSchemaPath`, I feel it will be left alone if we somehow decide to refactor all the extension schema path to something else. And there is probably somewhere else use the same purpose identifier "extensions" , if we can create or find existing identifier that represent "extensions" path then it will be ideal.

Reply:
For the first question, are you talking about we should make "extensions" as a constant？I could update it to use constant instead. "extensions" directory is like "pegasus" directory, we defined how the schema directory will be structured. Also we provide the config to customers, if they prefer to use other directory for extension schemas. I think what you said should not be a problem here.

Reply:
I checked PegasusPlugin, we defined all the "getDirectoryPath" methods in this way, e.g pegasus, idl, snapshot etc. For coding style consistent purpose, I would prefer to leave it as-is. We may need to clean the code to use constants all over the places, that should be in a separate PR.